### PR TITLE
MC-13271 Add create instance doc

### PR DESCRIPTION
**Pull request by razine-bensari**

### CODE REVIEW

File: source/includes/aws/_instances.md [hunk 1]
```diff
@@ -139,6 +139,79 @@ Attributes | &nbsp;
 `threadsPerCore`<br/>>*int* | The number of threads per CPU core.
 
 
+<!-------------------- CREATE AN INSTANCE -------------------->
+
+#### Create an instance
+```shell
+curl -X DELETE \
```
Comment (dpmurray21) [inline source/includes/aws/_instances.md]: should be POST

File: source/includes/aws/_instances.md [hunk 2]
```diff
@@ -139,6 +139,79 @@ Attributes | &nbsp;
...
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances</code>
+
+Create a new instance in a given [environment](#administration-environments).
+
+Required | &nbsp;
+ ------- | -----------
+`name`<br/>*string* | The name of the instance.
+`region`<br/>*string* | The region where the instance will be deployed.
+`imageId` <br/>*string* | The imageId to be used deployed with the instance. Represents the unique identifier of an amazon machine image (AMI). The imageId needs to be compatible with the machine type. The imageId needs to be available in the given region of the instance.
+`instanceType`<br/>*string* | The instance type for the instance. Represents the specification of the instance (cpu, memory, storage). The instance type needs to be available in the given region of the instance.
```
Comment (dpmurray21) [inline source/includes/aws/_instances.md]:
```suggestion
`instanceType`<br/>*string* | The type of the instance. Represents the specification of the instance (cpu, memory, storage). The instance type needs to be available in the given region of the instance.
```

File: source/includes/aws/_instances.md (hunk 1 above)
Comment (razine-bensari) [reply source/includes/aws/_instances.md]: I am not a copy pasta master 😭

Comment (razine-bensari) [inline source/includes/aws/_instances.md]:
```suggestion
curl -X POST \
```

**Review by dpmurray21 — APPROVED** [review]

**Review by mcaniglia16 — APPROVED** [review]

**Pull request merged**